howso/react_aggregate.amlg

```diff
@@ -462,32 +470,97 @@
 			(assign (assoc conditioned_residuals (true) ))
 		)
 
+		;If we are not always recalculating metrics, then we need to check which metrics exist in cache by attempting to retrieve them
+		(if (and (not always_recalculate) (= conditioned_residuals (false)))
+			(seq
+				(assign
+					(assoc output
+						(call !GetFeatureDetailsStats (append
+							(keep
+								details
+								(append !supportedReactAggregateDetailsStats ["selected_prediction_stats"])
+							)
+							(assoc
+								feature_influences_action_feature feature_influences_action_feature
+								prediction_stats_action_feature prediction_stats_action_feature
+								robust_hyperparameters robust_hyperparameters
+								weight_feature weight_feature
+								confusion_matrix_min_count confusion_matrix_min_count
+							)
+						))
+					)
+				)
+
+				(accum (assoc
+					existing_cached_stats
+						(values (map
```

Comment:
This (map) looks like it returns a list, so (values) may be unnecessary.

Reply:
looks like it becomes a list of list if i dont use values